.travis.yml

```diff
@@ -1,16 +1,37 @@
-language: java
-
-# This is essential lest the script is ignored and Travis just runs gradle.
-install: true
-
-script: ./.travis-build.sh
-
-sudo: false
-
-jdk:
-#  - openjdk7
-  - oraclejdk8
-
-
-git:
-  depth: 9
+{
```

Comment:
Can you convert this from this JSON format to the usual YML format?
The very long lines make the scripts hard to read. See the checker-framework `.travis.yml` file for examples.

Reply:
Done.